pages/Gold-Dust.md

```diff
@@ -8,6 +8,10 @@ Gold Dust has a ~11.11% chance of being extracted from [Sifted Ore](https://gith
 
 Additionally, crushing a [4-carat Gold Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Gold-Ingot#Gold-Ingot-4-Carat) in an [Electric Ingot Pulverizer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Pulverizer) produces one piece of Gold Dust.
 
+It is not possible to convert traditional (Smelted)  [Gold Ingot](https://minecraft.gamepedia.com/Gold_Ingot) into SlimeFun's Gold Dust/Ingot.
```

Comment:
Not really true, you can crush normal Gold Ingots using an Ore Crusher to get dust.
And Slimefun Gold Ingots can be used in workbench recipes like normal gold ingots.

Reply:
Thanks or the quick review!
Are you sure about this? (Vanilla Gold Ingots to Gold Dust in an Ore Crusher? I just tried it and it wasn't working. Also, what is a "SlimeFun Gold Ingot"? Any carat Ingot can be put into recipes which requires Vanilla Gold Ingot? Is this a config setting to enable?

Reply:
`Vanilla Gold Ingots to Gold Dust in an Ore Crusher? I just tried it and it wasn't working. `
Hmm, that's probably a bug then.

`Also, what is a "SlimeFun Gold Ingot"? Any carat Ingot can be put into recipes which requires Vanilla Gold Ingot? Is this a config setting to enable?`
Well, yeah any carat gold ingot can be used in workbenches if enabled via config, but newer versions have it turned on for the 4-carat gold ingot by default.

Reply:
OK, I will mention this in the Gold Ingot articles. I'd recommend to leave this as is and make a correction once that's in the codebase. I'm happy to make that correction in the code later and will submit a PR.